pkg/store/verifiable/store.go

```diff
@@ -50,17 +62,17 @@ type provider interface {
 }
 
 // New returns a new vc store
-func New(ctx provider) (*Store, error) {
+func New(ctx provider) (Store, error) {
```

Comment:
We need to return an implementation here. Where `StoreImplementation` is public.
e.g. `func New(ctx provider) (StoreImplementation, error)`
To make sure that implementation satisfies the interface do the following:
`var _ Store = (*StoreImplementation)(nil)`

Reply:
Fixed